[0m09:14:21.261879 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BC468D3770>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BC447E8190>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BC48AD0410>]}


============================== 09:14:21.264690 | 3454c13b-6643-4992-b5d1-7a2f42e48226 ==============================
[0m09:14:21.264690 [info ] [MainThread]: Running with dbt=1.10.11
[0m09:14:21.265159 [debug] [MainThread]: running dbt with arguments {'partial_parse': 'True', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True', 'version_check': 'True', 'cache_selected_only': 'False', 'target_path': 'None', 'warn_error': 'None', 'invocation_command': 'dbt init demo_dbt', 'log_format': 'default', 'use_colors': 'True', 'introspect': 'True', 'debug': 'False', 'printer_width': '80', 'fail_fast': 'False', 'no_print': 'None', 'quiet': 'False', 'use_experimental_parser': 'False', 'empty': 'None', 'log_cache_events': 'False', 'log_path': 'logs', 'indirect_selection': 'eager', 'profiles_dir': 'C:\\Users\\choco\\.dbt', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'write_json': 'True'}
[0m09:14:21.275040 [info ] [MainThread]: Creating dbt configuration folder at C:\Users\choco\.dbt
[0m09:14:21.276283 [debug] [MainThread]: Starter project path: C:\Users\choco\AppData\Local\Programs\Python\Python313\Lib\site-packages\dbt\include\starter_project
[0m09:14:21.292543 [info ] [MainThread]: 
Your new dbt project "demo_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m09:14:21.292848 [info ] [MainThread]: Setting up your profile.
[0m09:14:21.293435 [error] [MainThread]: Encountered an error:
Runtime Error
  No adapters available. Learn how to install an adapter by going to https://docs.getdbt.com/docs/connect-adapters#install-using-the-cli
[0m09:14:21.293973 [debug] [MainThread]: Command `dbt init` failed at 09:14:21.293891 after 0.11 seconds
[0m09:14:21.294198 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BC48AF6B10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BC48CF4950>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BC47987350>]}
[0m09:14:21.294436 [debug] [MainThread]: Flushing usage events
[0m09:14:21.468709 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m09:49:04.098620 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025FBD693770>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025FBB58C190>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025FBF894410>]}


============================== 09:49:04.101276 | 378edaa2-fe91-4903-9ae3-a00329674bb7 ==============================
[0m09:49:04.101276 [info ] [MainThread]: Running with dbt=1.10.11
[0m09:49:04.101700 [debug] [MainThread]: running dbt with arguments {'partial_parse': 'True', 'write_json': 'True', 'send_anonymous_usage_stats': 'True', 'warn_error': 'None', 'cache_selected_only': 'False', 'target_path': 'None', 'version_check': 'True', 'invocation_command': 'dbt debug', 'log_format': 'default', 'use_colors': 'True', 'introspect': 'True', 'debug': 'False', 'printer_width': '80', 'fail_fast': 'False', 'no_print': 'None', 'quiet': 'False', 'use_experimental_parser': 'False', 'empty': 'None', 'log_cache_events': 'False', 'indirect_selection': 'eager', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\choco\\.dbt', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True'}
[0m09:49:04.112083 [info ] [MainThread]: dbt version: 1.10.11
[0m09:49:04.112336 [info ] [MainThread]: python version: 3.13.2
[0m09:49:04.112536 [info ] [MainThread]: python path: C:\Users\choco\AppData\Local\Programs\Python\Python313\python.exe
[0m09:49:04.112724 [info ] [MainThread]: os info: Windows-11-10.0.26100-SP0
[0m09:49:05.714723 [info ] [MainThread]: Using profiles dir at C:\Users\choco\.dbt
[0m09:49:05.715155 [info ] [MainThread]: Using profiles.yml file at C:\Users\choco\.dbt\profiles.yml
[0m09:49:05.715481 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\choco\Documents\demo_dbt\dbt_project.yml
[0m09:49:05.716113 [info ] [MainThread]: adapter type: snowflake
[0m09:49:05.716368 [info ] [MainThread]: adapter version: 1.10.2
[0m09:49:05.716615 [info ] [MainThread]: Configuration:
[0m09:49:05.716848 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m09:49:05.717029 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m09:49:05.717207 [info ] [MainThread]: Required dependencies:
[0m09:49:05.717397 [debug] [MainThread]: Executing "git --help"
[0m09:49:05.736433 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--no-lazy-fetch]\n           [--no-optional-locks] [--no-advice] [--bare] [--git-dir=<path>]\n           [--work-tree=<path>] [--namespace=<name>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone      Clone a repository into a new directory\n   init       Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add        Add file contents to the index\n   mv         Move or rename a file, a directory, or a symlink\n   restore    Restore working tree files\n   rm         Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect     Use binary search to find the commit that introduced a bug\n   diff       Show changes between commits, commit and working tree, etc\n   grep       Print lines matching a pattern\n   log        Show commit logs\n   show       Show various types of objects\n   status     Show the working tree status\n\ngrow, mark and tweak your common history\n   backfill   Download missing objects in a partial clone\n   branch     List, create, or delete branches\n   commit     Record changes to the repository\n   merge      Join two or more development histories together\n   rebase     Reapply commits on top of another base tip\n   reset      Reset current HEAD to the specified state\n   switch     Switch branches\n   tag        Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch      Download objects and refs from another repository\n   pull       Fetch from and integrate with another repository or a local branch\n   push       Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m09:49:05.736673 [debug] [MainThread]: STDERR: "b''"
[0m09:49:05.736905 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m09:49:05.737143 [info ] [MainThread]: Connection:
[0m09:49:05.737368 [info ] [MainThread]:   account: gkzdcdb-lo63426
[0m09:49:05.737557 [info ] [MainThread]:   user: ivanivanov1995
[0m09:49:05.737733 [info ] [MainThread]:   database: SNOWFLAKE_SAMPLE_DATA
[0m09:49:05.738038 [info ] [MainThread]:   warehouse: SNOWFLAKE_LEARNING_WH
[0m09:49:05.738234 [info ] [MainThread]:   role: ACCOUNTADMIN
[0m09:49:05.738409 [info ] [MainThread]:   schema: TPCDS_SF100TCL
[0m09:49:05.738583 [info ] [MainThread]:   authenticator: None
[0m09:49:05.738747 [info ] [MainThread]:   oauth_client_id: None
[0m09:49:05.739009 [info ] [MainThread]:   query_tag: None
[0m09:49:05.739334 [info ] [MainThread]:   client_session_keep_alive: False
[0m09:49:05.739679 [info ] [MainThread]:   host: None
[0m09:49:05.739988 [info ] [MainThread]:   port: None
[0m09:49:05.740278 [info ] [MainThread]:   proxy_host: None
[0m09:49:05.740554 [info ] [MainThread]:   proxy_port: None
[0m09:49:05.740803 [info ] [MainThread]:   protocol: None
[0m09:49:05.741054 [info ] [MainThread]:   connect_retries: 0
[0m09:49:05.741270 [info ] [MainThread]:   connect_timeout: 10
[0m09:49:05.741465 [info ] [MainThread]:   retry_on_database_errors: True
[0m09:49:05.741651 [info ] [MainThread]:   retry_all: False
[0m09:49:05.741836 [info ] [MainThread]:   insecure_mode: False
[0m09:49:05.742014 [info ] [MainThread]:   reuse_connections: True
[0m09:49:05.742280 [info ] [MainThread]:   s3_stage_vpce_dns_name: None
[0m09:49:05.742746 [info ] [MainThread]: Registered adapter: snowflake=1.10.2
[0m09:49:05.925792 [debug] [MainThread]: Acquiring new snowflake connection 'debug'
[0m09:49:07.195246 [debug] [MainThread]: Using snowflake connection "debug"
[0m09:49:07.195552 [debug] [MainThread]: On debug: select 1 as id
[0m09:49:07.195752 [debug] [MainThread]: Opening a new connection, currently in state init
[0m09:49:10.297968 [debug] [MainThread]: SQL status: SUCCESS 1 in 3.103 seconds
[0m09:49:10.298667 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m09:49:10.299079 [info ] [MainThread]: [31m1 check failed:[0m
[0m09:49:10.299419 [info ] [MainThread]: Project loading failed for the following reason:
 project path <C:\Users\choco\Documents\demo_dbt\dbt_project.yml> not found

[0m09:49:10.299982 [debug] [MainThread]: Command `dbt debug` failed at 09:49:10.299910 after 6.31 seconds
[0m09:49:10.300183 [debug] [MainThread]: Connection 'debug' was left open.
[0m09:49:10.300348 [debug] [MainThread]: On debug: Close
[0m09:49:10.402076 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025FE0FD1BA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025FE0C31490>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025FDFA27CE0>]}
[0m09:49:10.402338 [debug] [MainThread]: Flushing usage events
[0m09:49:11.191302 [debug] [MainThread]: An error was encountered while trying to flush usage events
